.gitignore

```diff
@@ -0,0 +1,14 @@
+**
+!flathub.json
```

Comment:
a better .gitignore is opt-out not opt-in.

Reply:
Done